modules/um_callerid.c

```diff
@@ -63,6 +63,16 @@ um_callerid_modinit(void)
 		return -1;
 	}
 
+	user_modes['@'] = find_umode_slot();
```

Comment:
Can we... use something other than `@` for this mode? There are 52 alphabetical umodes, surely we haven't used all of them already.

Reply:
Yes. I figured there was going to be bikeshedding about the mode character whatever I used, so went with something I know everyone will hate to make sure it happens early.